muddy_macros/src/internal.rs

```diff
@@ -115,10 +122,10 @@ fn build_env_cipher_block(
     quote! {
         static #key_ident: Lazy<Key> = Lazy::new(|| {
             let Some(var) = std::env::var_os(#env_ident) else {
-                panic!()
+                panic!("Need to set {} env variable", #env_ident);
             };
-            let Ok(bytes) = <[u8; 32]>::from_hex(var.as_bytes()) else {
-                panic!()
+            let Ok(bytes) = <[u8; 32]>::from_hex(var.as_encoded_bytes()) else {
+                panic!("Can't get bytes from {} env variable, secret key needs to be a hex string with 64 symbols length.", #env_ident)
```

Comment:
```suggestion
                panic!()
```

Reply:
Found a way to keep debug information for developer using `#[cfg(debug_assertions)]` and remove it for release using `#[cfg(not(debug_assertions))].